### PR TITLE
Add support for ReCaptcha in addition to HCaptcha

### PR DESCRIPTION
**Pull request by jvyden**

### CODE REVIEW

File: ProjectLighthouse/Extensions/RequestExtensions.cs [hunk 1]
```diff
@@ -15,7 +15,21 @@ namespace LBPUnion.ProjectLighthouse.Extensions;
 
 public static class RequestExtensions
 {
-
+    static RequestExtensions()
+    {
+        Uri captchaUri = ServerConfiguration.Instance.Captcha.Type switch
+        {
+            CaptchaType.HCaptcha => new Uri("https://hcaptcha.com"),
+            CaptchaType.ReCaptcha => new Uri("https://www.google.com/recaptcha/api/"),
```
Comment (Slendy) [inline ProjectLighthouse/Extensions/RequestExtensions.cs:23]: Does it matter that one has a trailing slash and the other doesn't?

**Review by Slendy — APPROVED** [review]
Looks good to me.

File: ProjectLighthouse/Extensions/RequestExtensions.cs (hunk 1 above)
Comment (jvyden) [reply ProjectLighthouse/Extensions/RequestExtensions.cs:23]: In testing it does not.

Comment (Slendy) [reply ProjectLighthouse/Extensions/RequestExtensions.cs:23]: Should be fine then.

Comment (jvyden) [reply ProjectLighthouse/Extensions/RequestExtensions.cs:23]: Alright, merging then.

**Pull request merged**